resolution-buster.js

```diff
@@ -1,8 +1,8 @@
 /*eslint no-console: 0*/
```

Comment:
**suggestion**: it looks like this file forces consuming packages to use n-ui-foundation's dependencies transitively? that seems like a very bad idea, and shouldn't be necessary with npm (not sure it was with bower either) so I say we delete this file entirely